### ISSUE TITLE
Compact format option is ignored

### ISSUE THREAD

**Issue by laughedelic** [terraform-linters/tflint#1460]
## Introduction

It seems like `-f compact` formatting is ignored. It outputs the same as `default`. Other options like `-f json` or `-f sarif` seem to work fine.

## Expected Behavior

One-line per error output when using `tflint -f compact`

## Actual behavior

```console
►  tflint --init
Plugin `google` is already installed

►  tflint -f compact
Failed to load configurations; example.tf:3,16-4,1: Invalid expression; Expected the start of an expression, but found an invalid expression token.:

Error: Invalid expression

  on example.tf line 3, in module "example":
   3:   version    =
   4:   project_id = var.project_id

Expected the start of an expression, but found an invalid expression token.
```

As you can see it prints full output, same as with `-f default`.

## Step to Reproduce

Example file in the current directory: `example.tf`:

```hcl
module "example" {
  source     = "..."
  version    = 
  project_id = var.project_id
}
```

## Additional Context

```console
►  tflint -v
TFLint version 0.39.1
+ ruleset.google (0.18.0)
►  terraform -v
Terraform v1.2.6
on darwin_arm64
```

**Comment by bendrucker** [comment]
The compact format does not reformat _errors_ when loading configuration.
